### PR TITLE
Whiteboard component

### PR DESCRIPTION
**Pull request by Jigar-bhaliya**

### CODE REVIEW

File: bigbluebutton-html5/imports/ui/components/WhiteboardWrapper/WhiteboardWrapperContainer.jsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+import Auth from '/imports/ui/services/auth';
+import Users from '/imports/api/users';
+import PresentationAreaService from '/imports/ui/components/presentation/service';
+import WhiteboardWrapperView from './WhiteboardWrapperView';
+
+const ROLE_VIEWER = Meteor.settings.public.user.role_viewer;
+
+const WhiteboardWrapperContainer = ({
+  userIsPresenter, multiUser, podId, isViewer, whiteboardId,
```
Comment (piyushdhamecha) [inline bigbluebutton-html5/imports/ui/components/WhiteboardWrapper/WhiteboardWrapperContainer.jsx]: multi line

File: bigbluebutton-html5/imports/ui/components/WhiteboardWrapper/WhiteboardWrapperContainer.jsx [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  }).role === ROLE_VIEWER;
+
+  const defaultSlide = PresentationAreaService.getCurrentSlide(podId);
+
+  return {
+    userIsPresenter: PresentationAreaService.isPresenter(podId),
+    multiUser: PresentationAreaService.getMultiUserStatus(defaultSlide && defaultSlide.id),
+    podId,
+    isViewer,
+    whiteboardId: defaultSlide.id,
```
Comment (piyushdhamecha) [inline bigbluebutton-html5/imports/ui/components/WhiteboardWrapper/WhiteboardWrapperContainer.jsx]: can we order the key alphabetically?

File: bigbluebutton-html5/imports/ui/components/WhiteboardWrapper/WhiteboardWrapperView.jsx [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+import CursorWrapperContainer from '/imports/ui/components/presentation/cursor/cursor-wrapper-container/container';
+import AnnotationGroupContainer from '/imports/ui/components/whiteboard/annotation-group/container';
+
+const WhiteboardWrapperView = ({ whiteboardId }) => (
+  <div className="absolute w-full h-full z-50">
+    <svg
+      data-test="whiteboard"
+      width="100"
+      height="100"
+        // ref={(ref) => { if (ref != null) { this.svggroup = ref; } }}
```
Comment (piyushdhamecha) [inline bigbluebutton-html5/imports/ui/components/WhiteboardWrapper/WhiteboardWrapperView.jsx]: are we going to use it?

File: bigbluebutton-html5/imports/ui/components/WhiteboardWrapper/WhiteboardWrapperContainer.jsx (hunk 1 above)
Comment (Jigar-bhaliya) [reply bigbluebutton-html5/imports/ui/components/WhiteboardWrapper/WhiteboardWrapperContainer.jsx]: fixed

File: bigbluebutton-html5/imports/ui/components/WhiteboardWrapper/WhiteboardWrapperContainer.jsx (hunk 2 above)
Comment (Jigar-bhaliya) [reply bigbluebutton-html5/imports/ui/components/WhiteboardWrapper/WhiteboardWrapperContainer.jsx]: fixed

File: bigbluebutton-html5/imports/ui/components/WhiteboardWrapper/WhiteboardWrapperView.jsx (hunk 3 above)
Comment (Jigar-bhaliya) [reply bigbluebutton-html5/imports/ui/components/WhiteboardWrapper/WhiteboardWrapperView.jsx]: removed

File: bigbluebutton-html5/imports/ui/components/WhiteboardWrapper/WhiteboardWrapperContainer.jsx [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+  const svgRef = useRef(null);
+
+  const { width, height } = useContainerDimensions(divRef);
+
+  const MAX_Z_INDEX = (2 ** 31) - 1;
+
+  const baseName = Meteor.settings.public.app.cdn + Meteor.settings.public.app.basename;
+
+  let cursor;
+  if (!userIsPresenter) {
```
Comment (piyushdhamecha) [inline bigbluebutton-html5/imports/ui/components/WhiteboardWrapper/WhiteboardWrapperContainer.jsx:56]: Add empty line before

File: bigbluebutton-html5/imports/api/annotations/server/methods/undoAnnotation.js [hunk 5]
```diff
@@ -16,5 +17,6 @@ export default function undoAnnotation(whiteboardId) {
     whiteboardId,
   };
 
+  Logger.info(`channel - ${CHANNEL} event name- ${EVENT_NAME} meeting Id - ${meetingId} userid - ${requesterUserId} payload - ${JSON.stringify(payload)}`);
```
Comment (piyushdhamecha) [inline bigbluebutton-html5/imports/api/annotations/server/methods/undoAnnotation.js]: Do we need this logger?

**Pull request merged**